CONTRIBUTING.md

```diff
@@ -0,0 +1,115 @@
+Contributing to the ITK Software Guide with Git and GitHub
+----------------------------------------------------------
+
+<!-- Uncomment these lines after the ITK repository has finished migration to GitHub
+
+The contribution process mirrors the [ITK contribution process](https://github.com/InsightSoftwareConsortium/ITK/blob/master/CONTRIBUTING.md) with the exception of the clone url:
+
+   https://github.com/InsightSoftwareConsortium/ITKSoftwareGuide.git
+-->
+
+A basic knowledge of [Git] underpins the contribution process. A concise
+but handy guide to [Git] can be found
+[here](http://rogerdudler.github.io/git-guide/).
+
+
+Contributing Patches with Git and GitHub
+----------------------------------------
+
+To obtain a copy of the project sources, run
+
+```sh
+  git clone https://github.com/InsightSoftwareConsortium/ITKSoftwareGuide.git
+```
+
+After cloning the repository on your local machine for the first time, it is
+necessary to provide Git with your authorship identity details. The
+`ITKSoftwareGuide/Utilities/SetupForDevelopment.sh` script does just that, and
+it provides an opportunity to change other basic settings.
+
+Before you start making changes, a new branch needs to be created with a name
+that describes the change's topic: `git checkout -b topic-name`.
+
+Next, use the editor of your choice to make the changes to the file(s).
+
+When you are satisfied with your changes, use `git add -- fileName` to stage the
+file to commit.
+
+When you run `git commit`, you will be presented with a [VIM] editor window to
+type your commit message. The commit message must start with a one of the
+following standard prefixes:
+
+  * `BUG`: Fix for runtime crash or incorrect result
+  * `COMP`: Compiler error or warning fix
+  * `DOC`: Documentation change
+  * `ENH`: New functionality
+  * `PERF`: Performance improvement
+  * `STYLE`: No logic impact (indentation, comments)
+  * `WIP`: Work In Progress not ready for merge
+
+Following the prefix, separated by a colon and a space, type a brief one-line
+description for the patch. Type a more detailed description of the patch after
+a blank line.
+
+Submit the patch for peer review by running the `git review-push` command.
+This will provide a summary message that inclues a URL to view the changes. If
+you are logged into [GitHub], there will also be a green button to create a
+[Pull Request].
+
+The automated continuous integration builds will build your patch. If the
+*CircleCI* build fails, click the CircleCI and CDash `Details` link to find
+additional information on why the build failed. If the CircleCI build
+succeeds, click on the CDash status `Details` link. Click the yellow package
+icon to obtain links to the rendered Software Guide PDF's for inspection.
+
+Peers will review the patch, and you will receive emails with review comments.
+Reply to the comments via email or the GitHub web interface. Follow-up with
```

Comment:
I'd remove the `via email` part, or modify the message. Although we do it from time to time for messages that we consider that do not need to be in the topic history, I'd say that as a general rule, we should keep all comments in the GitHub web interface.

Reply:
Thanks for the review, @jhlegarreta .

Here I meant *reply-to-all* to the emails sent by GitHub, which will also show up on the GitHub web interface. But, I can see how it could be interpreted in a different way.  I will remove this sentence for clarity.

Could you please review and mark the PR as *Approved* when ready? The settings have changed on the repository so a patch must be Approved before merging.

Reply:
OK, I was not thinking about the `reply-to-all` possibility. Adding it but make no harm. So leaving at your discretion Matt.

Yes, I've noticed that merging is blocked until the PR gets some reviewer's approval.